src/Microsoft.DocAsCode.Build.Engine/PostProcessors/IncrementalPostProcessorsContext.cs

```diff
@@ -125,12 +126,13 @@ private bool GetCanIncremental()
                 var lastPostProcessorInfo = LastInfo.PostProcessorInfos[i];
                 if (!currentPostProcessorInfo.Equals(lastPostProcessorInfo))
                 {
-                    var message = $"Cannot support incremental post processors, because post processor info changed from last {lastPostProcessorInfo.ToJsonString()} to current {currentPostProcessorInfo.ToJsonString()}.";
+                    var message = $"Cannot support incremental post processing, because post processor info changed from last {lastPostProcessorInfo.ToJsonString()} to current {currentPostProcessorInfo.ToJsonString()}.";
                     Logger.LogVerbose(message);
                     return false;
                 }
             }
 
+            Logger.LogVerbose("Can support incremental post processing.");
```

Comment:
renamed to "IsIncremental"

---
In reply to: [95507816](https://github.com/dotnet/docfx/pull/1136#discussion_r95507816) [](ancestors = 95507816,95506929)